learn/advanced/faceted_search.md

```diff
@@ -0,0 +1,208 @@
+      "Toys & Games":1
+    },
+    "rating":{
+      "0":7,
+      "1":1,
+      "3.6":1,
+      "3.8":1,
+      "4":2,
+      "4.1":2,
+      "4.4":1,
+      "4.5":2,
+      "4.6":1,
+      "4.7":3,
+      "5":3
+    }
+  }
+  …
+}
+```
+
+`facetDistribution` contains an object for every given facet, in this case, `brand`, `category`, and `rating`. For each of these facets, there is another object containing all the different values and the count of matching documents. Note that zero values will not be returned: if there are no `led` products under the `Electronics` category, it is not displayed.
+
+::: note
+By default, `facets` returns a maximum of 100 facet values for each faceted field. You can change this value using the `maxValuesPerFacet` property of the [`faceting` index settings](/reference/api/settings.md#faceting).
+:::
+
+### Facet stats
+
+When using the `facets` parameter, any matching documents with facets containing numeric values are displayed in a `facetStats` object. `facetStats` contains the numeric minimum (`min`) and maximum (`max`) values per facet for all documents matching the search query. This can be used to create a range slider component allowing users to select a range of values for a facet.
+
+If none of the matching documents have a numeric value for a facet, that facet is not included in the `facetStats` object.
+
+Meilisearch ignores string values like `"21"` when computing `facetStats`.
+
+The following response shows the `facetStats` when searching for `led`:
+
+```json
+{
+  …
+  "facetStats":{
+    "rating":{
+      "min":0.0,
+      "max":5.0
+    }
+  }
+  …
+}  
+```
+
+Since `rating` was the only numeric facet in our example, it is returned in the `facetStats` object.
+
+## Facet types
+
+### Single select facets
+
+Single-select or conjunctive facets use the `AND` logical operator. They only allow the selection of a single value per facet.
+
+Suppose your `ecommerce` index contains a facet for `brand` with the values `BrandA`, `BrandB`, and `BrandC`. Single select facets will allow you to view products from one brand at a time. If you want to view `BrandA` and `BrandB` products, you will have to try both searches individually. If you want to further narrow down results using the `reviews` facet, you would need to do so separately for both brands.
+
+The image below shows single-select facets in action. When the user searches for `led` and selects `Sunlite`, other brands are no longer visible.
+
+![Meilisearch demo for single-select facets searching for 'LED'](/faceted-search/facets-single-select.png)
+
+The code sample below shows the query for the search mentioned above:
+
+<CodeSamples id="faceted_search_conjunctive_facets_1" />
+
+### Multi-select facets
+
+Multi-select or disjunctive facets use the `OR` logical operator. They allow users to choose multiple options within a facet so they don’t have to perform more than one search to find the products they’re looking for.
```

Comment:
Can we also add a line telling that a logical `AND` is done **between the categories**?

Reply:
Done ✅